x-pack/plugins/cases/server/client/alerts/types.ts

```diff
@@ -7,17 +7,7 @@
 
 import { CaseStatuses } from '../../../common/api';
 import { AlertInfo } from '../../common';
-
-interface Alert {
-  id: string;
-  index: string;
-  destination?: {
```

Comment:
We weren't referencing these fields. We're using a lodash `get` so we don't actually need the explicit fields defined in the type.